persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/DsePersistence.java

```diff
@@ -440,14 +440,14 @@ private DseConnection() {
     @SuppressWarnings("RxReturnValueIgnored")
     private DseConnection(@Nullable ClientInfo clientInfo, ClientState clientState) {
       super(clientInfo);
-      this.clientState = clientState;
+      this.clientState = ClientStateWithIsExternal.from(clientState);
```

Comment:
I confirmed that this breaks the interceptors that rely on `instanceof` checks. Pushing another commit to fix that.